styles.css

```diff
@@ -0,0 +1,343 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;500;600;700;800;900&display=swap");
+
+* {
+  margin: 0;
+}
+
+:root {
+  height: 100%;
+  --primary-text: #000;
+  --secondary-text: #666;
+  --primary-bg: #fff;
+  --secondary-bg: #f7f9fa;
+  --tertiary-bg: #e0f3ff;
+  --hover-bg: #e7f5fd;
+  --accent: #ff1f13;
+  --button: #39a2f2;
```

Comment:
prefixing with `color-` will make the variables more clear. `--button` being used at some place would be very confusing.